### ISSUE TITLE
Upgrade to v0.20.147

### ISSUE THREAD

**Issue by giedriusStan** [Sabuto/hassio-jackett#17]
A warning message appears at the bottom of web UI:

Your version of Jackett v0.12.999.0 is very old. Multiple indexers are likely to fail when using an old version. Update to the latest version of Jackett.

**Comment by sabuto** [comment]
I will bump the version next time I get to the computer and do a bit of testing thanks:)

**Comment by lbouriez** [comment]
I forked and updated it there if needed: https://github.com/lbouriez/addon-jackett

**Comment by sabuto** [comment]
> I forked and updated it there if needed: https://github.com/lbouriez/addon-jackett

thanks for this, does this work alright? i will test it tomorrow for sure and if so i will accept a PR

**Comment by lbouriez** [comment]
I am testing it right now, I changed all the pipelines also to work with the way works the official addons.
I can create a PR but it will break your pipelines I guess. let me know.

**Comment by sabuto** [comment]
When you say pipelines do you mean the github actions?

**Comment by lbouriez** [comment]
Yep.

**Comment by sabuto** [comment]
ok cool, i'll look opver them, does it publish it to github containers now?

**Comment by lbouriez** [comment]
Yes, I copied all the action of the officials addons, the way they build and publish to the hassio repo.
Have a look, if you are ok with it I can create the PR.
Also, I had an error using the indexer url that is given into jackett (the one that contains ingress) instead I had to use the one with the port 8889 like `http://ha_ip:8889/api/v2.0/indexers...` The issue was that using the one with ingress needed an authentication and so it was forbidden... I dont know how works the hassio addons enough to fix it :)

**Comment by sabuto** [comment]
Kk ill have a look through, yeah I haven't had much to do with hassle addons for a while so I'll read through the documentation

**Comment by cdmn1** [comment]
Is there any up-to-date/maintained version of a Jackett addon? 
This one and lbouriez version both seem abandoned (can't even raise an issue in lbouriez's repo) and all internet searching leads me here.
